modules/ROOT/pages/component-authorization-using-spring-security.adoc

```diff
@@ -0,0 +1,71 @@
+= Component Authorization Using Spring Security
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: authorization, spring, spring security
+
+Learn how to configure authorization over Mule components using Spring Security features, such as the Authentication Manager or configuring secure components in asynchronous systems, so that users with different roles can only invoke certain operations.
+
+
+== Configure Spring Security Authentication Manager
+
+The Authentication Manager is responsible for passing requests through a chain of Authentication Provider objects. To secure calls to flow components, you must add the configured `authenticationManager` to the application context.
```

Comment:
Are "Authentication Manager" and "Authentication Provider" the names of Spring Security products or features? I tried to look them up but found only things like "AuthenticationManager" and "ProviderManager."

Reply:
"the configured `authenticationManager`" looks, in the syntax below, like `authenticationManager` is just a name for the authentication manager component. I'm not sure how to recommend term presentation without knowing for sure if the terms I've seen so far are official products or features, or general components, or alias names in syntax.

Reply:
I am further confused when I get to the two subtopics.
What are the two parts of Spring Security that we are documenting? I see "authentication manager," "authentication provider," "security property," "security provider," "security manager," and "delegate security provider."